inputs/values/dockers.json

```diff
@@ -33,4 +32,4 @@
   "sv_utils_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-utils:2023-03-16-v0.27-beta-906c6272",
   "gq_recalibrator_docker": "us.gcr.io/broad-dsde-methods/tbrookin/gatk:0a7e1d86f",
   "str": "us.gcr.io/broad-dsde-methods/gatk-sv/str:2023-05-23-v0.27.3-beta-e537bdd6"
-}
\ No newline at end of file
+}
```

Comment:
I think the action should be able to build the `sv_*` docker images; if you agree, I guess it would be better to change only `gatk_docker` and `gatk_docker_concordance` images in this file. 

Additionally, we should also re-tag this image for `dockers_azure.json`.

Reply:
I see your point about the `sv_` images and that it's not necessary and maybe pollutes the git history a bit. This is how I usually keep track of my dockers for testing and I think there's a case to be made for keeping a record of what dockers were used for testing in the PR. If it's okay with you, I'd prefer to leave these changes in, unless it creates an actual problem with the github action.

I've also removed `gatk_docker_concordance` and retagged `gatk_docker` in `dockers_azure.json`

Reply:
Following up on our discussion, I've reverted changes to `sv_` dockers to avoid potential synchronization issues with the automated docker builds and will do so going forward.

Reply:
Thanks for reverting this!